### PR TITLE
Docker image

### PR DESCRIPTION
**Pull request by romansaul**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,19 +1,32 @@
...
+COPY /docker/npm-license-checker.sh /usr/bin/npm-license-checker
 
 RUN chmod +x /usr/bin/pivio
 
+## License checker does not support getting started from a different directory
+# Getting prerequisites to npm:
+RUN apt-get update -yq && apt-get upgrade -yq && apt-get install -yq curl git nano gnupg2
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash && apt-get install -yq nodejs build-essential
+RUN apt-get install npm
+RUN npm install license-checker
```
Comment (rwe17) [inline docker/Dockerfile]: Dit you forgot to checkit your npm-license-checker.sh shell script

File: docker/Dockerfile [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
 
 COPY /build/libs/pivio.jar /pivio.jar
 COPY /docker/pivio.sh /usr/bin/pivio
+COPY /docker/npm-license-checker.sh /usr/bin/npm-license-checker
 
 RUN chmod +x /usr/bin/pivio
 
+## License checker does not support getting started from a different directory
+# Getting prerequisites to npm:
+RUN apt-get update -yq && apt-get upgrade -yq && apt-get install -yq curl git nano gnupg2
```
Comment (rwe17) [inline docker/Dockerfile]: I've changed this to one RUN command in order to reduce the total size of docker image.

File: docker/dockerbuild.sh [hunk 3]
```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# --- Option processing --------------------------------------------
+display_usage() { 
+    echo "Builds the docker container for the pivio-client" 
+    echo -e "\nUsage:\n$0 [options]\n" 
+    echo -e "  Options: -b           builds the pivio-client.jar artifact before building the docker container"
```
Comment (romansaul) [inline docker/dockerbuild.sh]: Why does the dockerbuild.sh also build the pivio.jar with gradle? There is already a build configured in our Jenkins and documented in README.md of the pivio-client repo.

File: docker/pivio.sh [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+# ------------------------------------------------------------------
+
+
+CONTAINER_NAME="leanix/pivio-client"
+
+
+# --- Option processing --------------------------------------------
+display_usage() { 
+    echo "Shell script used to wrap the pivio client java call within the docker container." 
+    echo -e "\nUsage:\ndocker run -it [docker run options] ${CONTAINER_NAME} pivio [pivio client options]\n"
```
Comment (romansaul) [inline docker/pivio.sh]: Param -it not needed here since this is not an interactive script. Also see further below.

**Pull request merged**